web/package.json

```diff
@@ -17,6 +17,9 @@
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "eslint-plugin-vue": "^4.4.0",
+    "pug": "^2.0.3",
```

Comment:
I agree with the choice to use pug. However, we should keep in mind that `eslint-plugin-vue` doesn't currently lint pug templates in Vue single file components.